src/styles/ConstructibleStyleSheets.ts

```diff
@@ -0,0 +1,7 @@
+import {css, unsafeCSS} from "lit-element";
+import globalCss from "./global.scss";
+import homeCss from "../views/vmd-home.view.scss";
+
+
+export const CSS_Global = css`${unsafeCSS(globalCss)}`
+export const CSS_Home = css`${unsafeCSS(homeCss)}`
```

Comment:
Ou simplement `unsafeCSS(…)` sans la `css` tagged literal.
```suggestion
export const CSS_Global = unsafeCSS(globalCss)`
export const CSS_Home = unsafeCSS(homeCss)`
```

Reply:
La "bonne pratique" semble en fait être d'utiliser la tagged literal: https://github.com/CovidTrackerFr/vitemadose-front/pull/141#issuecomment-832644369